gordon/interfaces.py

```diff
@@ -120,3 +120,46 @@ class IPublisherClient(IGenericPlugin):
         Args:
             event_msg (IEventMessage): Message ready to be sent to destination.
         """
+
+
+class IMetricRelay(Interface):
```

Comment:
What do you see the role of the metrics relay being in Gordon?  I've found limited use for it from Shumway and I'm concerned about codifying the design elsewhere. My thinking is that for our use case it makes sense to have an object that stores the socket information to ffwd, and an `emit` type method. For other users I can see the value of a some sort of internal data store of metrics and a `pull` method. I do also have a concern about trying to force two different use cases (socket store, emit) and (metrics store, pull) into the same abstraction, making it harder to understand both, since there seems to be very little overlap.